src/components/DashboardAdmin/index.jsx

```diff
@@ -2,81 +2,50 @@ import { useState } from "react";
 import { useLoaderData } from "react-router-dom";
 import CardsUsers from "../CardsUsers";
 import FormUsers from "../FormUsers";
+import TableUsers from "../TableUsers";
 import { getRoles } from "../../api/roles";
+import './index.scss'
+
 
 const DashboardAdmin = () => {
 
   const { roles } = useLoaderData();
 
-  console.log(roles)
-
   const [showForm, setShowForm] = useState(false);
 
   const handleShowForm = () => {
     setShowForm(true)
   }
 
+  const handleCloseForm = () => {
+    setShowForm(false)
```

Comment:
If you are using semicolon in a js/ts file, do not forget using it in whole the file (in each end of line) for coding formatting, this is not a blocker, but it's visually consistent.